samples/interactive-tutorials/events/setup_events/setup_cleanup_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
```

Comment:
Let's remove the tests for the setup scripts as they were used one-off and may break the test data.

Reply:
ok. done